config/app.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'name' => 'Integration Commons',
```

Comment:
The library is not really a laravel project. Is there really a need for config/app.php?